packages/integration/api-report.md

```diff
@@ -757,4 +758,8 @@ export class SingleInstanceGithubCredentialsProvider
   static create: (config: GithubIntegrationConfig) => GithubCredentialsProvider;
   getCredentials(opts: { url: string }): Promise<GithubCredentials>;
 }
+
+// Warnings were encountered during analysis:
```

Comment:
You'll need to fix this as this is what is causing the build to fail

Reply:
I missed that, will have a look 👍

Reply:
The warnings should be gone now :+1: